File: PieGallery/app/src/main/java/com/deadely/piegallery/view/main/IMainContract.java
package com.deadely.piegallery.view.main;

import com.deadely.piegallery.base.IBaseMvpPresenter;
import com.deadely.piegallery.base.IBaseMvpView;

public interface IMainContract {
    interface View extends IBaseMvpView {
    }

    interface Presenter extends IBaseMvpPresenter<View> {
    }

}

File: PieGallery/app/src/main/java/com/deadely/piegallery/ui/main/IMainContract.kt
package com.deadely.piegallery.ui.main

import com.deadely.piegallery.base.IBaseMvpPresenter
import com.deadely.piegallery.base.IBaseMvpView

interface IMainContract {
    interface View : IBaseMvpView

    interface Presenter : IBaseMvpPresenter<View>
}
